patches/api/0391-Add-getDrops-to-BlockState-and-isPreferredTool-to-Bl.patch

```diff
@@ -0,0 +1,91 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: MelnCat <melncatuwu@gmail.com>
+Date: Fri, 12 Aug 2022 23:24:53 -0700
+Subject: [PATCH] Add getDrops to BlockState and isPreferredTool to BlockData
+
+
+diff --git a/src/main/java/org/bukkit/block/BlockState.java b/src/main/java/org/bukkit/block/BlockState.java
+index 10cbe71917bc32cca61748bcb0aa3395c554dbf8..83117796f303f8249a94afedd72cf1898b33c0ae 100644
+--- a/src/main/java/org/bukkit/block/BlockState.java
++++ b/src/main/java/org/bukkit/block/BlockState.java
+@@ -5,12 +5,16 @@ import org.bukkit.Location;
+ import org.bukkit.Material;
+ import org.bukkit.World;
+ import org.bukkit.block.data.BlockData;
++import org.bukkit.entity.Entity;
++import org.bukkit.inventory.ItemStack;
+ import org.bukkit.material.MaterialData;
+ import org.bukkit.metadata.Metadatable;
+ import org.jetbrains.annotations.Contract;
+ import org.jetbrains.annotations.NotNull;
+ import org.jetbrains.annotations.Nullable;
+ 
++import java.util.Collection;
++
+ /**
+  * Represents a captured state of a block, which will not change
+  * automatically.
+@@ -233,5 +237,34 @@ public interface BlockState extends Metadatable {
+      * @return true if collidable
+      */
+     boolean isCollidable();
++
++    /**
++     * Returns a list of items which would drop by destroying this block state.
```

Comment:
say the list is "immutable" and then make it immutable.

Can use the `@Unmodifiable` jet brains annotation too.

Do this for all 3 getDrops methods.